dotnet/samples/KernelSyntaxExamples/Program.cs

```diff
@@ -25,58 +25,58 @@ public static async Task Main()
         await Example_00_03_OpenApiSkill_PlayFab.RunAsync().SafeWaitAsync(cancelToken);
 
         // Run examples
-        await Example01_NativeFunctions.RunAsync().SafeWaitAsync(cancelToken);
```

Comment:
Can ignore changes to this file for the PR

Reply:
Sure, I will revert. But all the examples will run whenever we run the project, so these are not required and save time if we comment.